hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1228,6 +1230,27 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 		return fmt.Errorf("failed to reconcile controlplane operator deployment: %w", err)
 	}
 
+	// Reconcile operator PodMonitor
+	podMonitor := controlplaneoperator.PodMonitor(controlPlaneNamespace.Name, hcluster.Name)
```

Comment:
would this work if running HA or would we need a serviceMonitor? https://github.com/openshift/hypershift/pull/532

Reply:
Yeah that is my understanding, the advantage of a podMonitor over a serviceMonitor is mainly that you do not need to create a Service first: https://github.com/prometheus-operator/prometheus-operator/issues/3119#issuecomment-610445145